policyengine_us/parameters/gov/states/oh/tax/income/credits/adoption/minimum_amount.yaml

```diff
@@ -0,0 +1,12 @@
+description: Ohio issues the adoption credit of at least this amount.
+values:
+  2021-01-01: 1_500
+metadata:
+  unit: currency-USD
+  period: year
+  label: Ohio adoption credit minimum amount
+  reference:
+    - title: Ohio 2021 Instructions for Filing Original and Amended - Line 17 – Ohio Adoption Credit
+      href: https://tax.ohio.gov/static/forms/ohio_individual/individual/2021/pit-it1040-booklet.pdf#page=21
+    - title: Ohio Income - Individual Credits (Education, Displaced Workers & Adoption)
```

Comment:
cant use this as this is not a primary source - need legal code and 2022 tax form

Reply:
both yaml